src/App.js

```diff
@@ -68,8 +69,17 @@ class Root extends React.Component {
   componentDidUpdate(prevProps) {
     // When Ledger device loses connection or the app is closed
     if (this.props.ledgerClosed && !prevProps.ledgerClosed) {
-      hathorLib.wallet.lock();
-      this.props.history.push('/wallet_type/');
```

Comment:
In this wallet type screen we are showing an alert message if the ledger app was closed nowadays. With this change, we might never go to this screen and only to the choose wallet, so we should add this alert there as well.

Reply:
I didn't understand.